### PR TITLE
Remove WinHandler::connected method

### PR DESCRIPTION
**Pull request by pyroxymat**
Solves issue #547 

I am not sure how to verify if my code still runs on Mac and Windows since I use Linux for development.

### CODE REVIEW

File: druid/src/win_handler.rs [hunk 1]
```diff
@@ -606,9 +606,7 @@ impl<T: Data> WinHandler for DruidHandler<T> {
         //it's possible that we should move those calls to occur in connected?
```
Comment (cmyr) [inline druid/src/win_handler.rs]: we can also delete this comment while we're in here

Comment (pyroxymat) [reply druid/src/win_handler.rs]: Whoops that comment is indeed outdated. I removed it.

**Comment by cmyr** [comment]
huh, CI is in a weird state, I'm not really sure what's going on 😕

**Comment by cmyr** [comment]
I'm going to go ahead and merge this, I think ci is just borked.

**Pull request merged**